### PR TITLE
Exercise 22

### PR DESCRIPTION
**Pull request by DPS7991**

### CODE REVIEW

File: JS/Exercise_22/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+alert('Enter two numbers to see if the first is greater than the second');
+let firstNumber = prompt('Enter the first number');
+let secondNumber = prompt('Enter the second number');
```
Comment (chriztie) [inline JS/Exercise_22/index.js]: As a best practice, convert values to numbers before comparing them as numbers. In this case, the values returning from prompt are strings not numbers. Great job though. Don't lose the momentum! ⏩ 🥇

**Review by chriztie — CHANGES_REQUESTED** [review]

File: JS/Exercise_22/index.js (hunk 1 above)
Comment (DPS7991) [reply JS/Exercise_22/index.js]: Changes have been made!

**Comment by DPS7991** [comment]
Made the changes

**Review by chriztie — APPROVED** [review]
Good job 👌

**Pull request merged**